app/models/host/managed.rb

```diff
@@ -197,6 +197,8 @@ class Jail < ::Safemode::Jail
     validate :check_if_provision_method_changed, :on => :update, :if => Proc.new { |host| host.managed }
     validates :uuid, uniqueness: { :allow_blank => true }
   else
+    include Orchestration::ProgressReport
```

Comment:
There shouldn't be any orchestration modules present in Host::Managed when not using unmanaged mode. The need for these methods should be removed, or it should be in Host::Managed proper, or Host::Base if applicable.